src/util/upnp_quirks.cc

```diff
@@ -61,7 +61,7 @@ void Quirks::restoreSamsungBookMarkedPosition(const std::shared_ptr<CdsItem>& it
 {
     if ((pClientInfo->flags & QUIRK_FLAG_SAMSUNG_BOOKMARK_SEC) == 0 && (pClientInfo->flags & QUIRK_FLAG_SAMSUNG_BOOKMARK_MSEC) == 0)
         return;
-    auto positionToRestore = item->getBookMarkPos();
+    auto positionToRestore = item->getBookMarkPos().count();
```

Comment:
BookMarkPos can be either MS or seconds I think? How are you handling that?

Reply:
No idea about this one.